Automattic/wp-calypso
# Classic Editor: removes After the Deadline from the toolbar

#### Changes proposed in this Pull Request

Removes the _Proofread Writing_ button from the Classic Editor toolbar, along with all related files.

| Before | After |
| - | - |
| ![Screen Shot 2019-03-29 at 03 20 22](https://user-images.githubusercontent.com/233601/55213429-fe6bc080-51d1-11e9-97b9-939e74b91b69.png) | ![Screen Shot 2019-03-29 at 03 21 40](https://user-images.githubusercontent.com/233601/55213443-0a578280-51d2-11e9-8fcc-ccb17396ac94.png) |

#### Testing instructions



* Opt out of the Block Editor on any WordPress.com site.
* Edit or Create a Post or Page with the Classic Editor.
* Verify that there are no regressions with the editing experience.

Part of #28550
